client/src/locations-list-page-state/actions/locationsListAction.js

```diff
@@ -0,0 +1,11 @@
+import * as actionTypes from "./actionTypes";
+
+export const addLocation = (id,name, lat,lng) => {
+  return {
+    type: actionTypes.ADD_NEW_LOCATION,
```

Comment:
what if something values came null or undefined? there are no checks or fallback in here or in reducer

Reply:
In such cases, I'm returning existing state